index.html

```diff
@@ -49,8 +49,9 @@
       <div class="logo"></div>
       <div class="font28 slogan transition">«<span class="fontsemibold">de</span>centralized <span class="fontsemibold">CREDIT</span>»</div>
       <div class="description font28">Decred is an open and progressive cryptocurrency with a system of community-based governance integrated into its blockchain.</div>
-      <a class="borderradius4 buttonlearnmore dropshadow font14 fontbold transition w-button" href="#overview">Learn more about Decred</a>
-      <a class="borderradius4 buttongetstarted dropshadow font14 fontbold transition w-button" href="#guide">Get started</a>
+      <a class="button download-wallet w-button" href="#guide">Download Wallet<span class="button-description">For Mac OS X</span></a>
```

Comment:
Well if the OS detection stuff is coming in another commit, I'd like this changed to something generic like s/For Mac OS X/For Your Computer/